AUDIO/HintScriptAudioBank.md

```diff
@@ -5,15 +5,17 @@ ns: AUDIO
 
 ```c
 // 0xFB380A29641EC31A 0x41FA0E51
-Any HINT_SCRIPT_AUDIO_BANK(Any p0, int p1);
+cs_type(Any) BOOL HINT_SCRIPT_AUDIO_BANK(cs_type(Any) char* bankName, cs_type(int) BOOL bOverNetwork);
```

Comment:
This native has 3 param actually:

- The bank name
- If network
- Player bits (int)

Reply:
You can't add native arguments as it will break the comparability of the type.